apps/studio/components/interfaces/Storage/AnalyticBucketDetails/NamespaceRow.tsx

```diff
@@ -65,66 +206,154 @@ export const NamespaceRow = ({
-      <TableCell className="text-right">
-        <div className="flex gap-2 justify-end">
-          <ButtonTooltip
-            type={missingTables.length > 0 ? 'primary' : 'default'}
-            icon={<RefreshCw className="h-3 w-3" />}
-            loading={isImportingForeignSchema || isLoadingNamespaceTables}
-            onClick={() => (schema ? rescanNamespace() : setImportForeignSchemaShown(true))}
-            disabled={missingTables.length === 0}
-            tooltip={{ content: { text: scanTooltip } }}
-          >
-            Sync
-          </ButtonTooltip>
-          {schema ? (
-            <Button asChild type="default" icon={<SquareArrowOutUpRight />}>
-              <a
-                target="_blank"
-                rel="noreferrer"
-                href={`${BASE_PATH}/project/${project?.ref}/editor?schema=${schema}`}
-              >
-                Open in Table Editor
-              </a>
-            </Button>
-          ) : (
-            <ButtonTooltip
-              type="default"
-              icon={<SquareArrowOutUpRight />}
-              disabled
-              tooltip={{
-                content: { text: 'There are no tables connected.' },
-              }}
+    <Card>
+      <CardHeader className="flex flex-row justify-between items-center px-4 py-5 space-y-0">
+        <CardTitle className="text-sm font-normal font-sans normal-case leading-none flex flex-row items-center gap-x-1">
+          <TooltipProvider>
+            <Tooltip>
+              <TooltipTrigger asChild>
+                <span className="flex flex-row items-center gap-x-1 text-foreground-lighter">
+                  {sourceType === 'direct' ? namespace : 'public'}
+                  <ChevronRight size={12} className="text-foreground-muted" />
+                </span>
+              </TooltipTrigger>
+              <TooltipContent>
+                <p>Source {sourceType === 'direct' ? 'namespace' : 'schema'}</p>
+              </TooltipContent>
+            </Tooltip>
+          </TooltipProvider>
+          {validSchema && (
+            <TooltipProvider>
+              <Tooltip>
+                <TooltipTrigger
+                  asChild
+                  className={tables.length === 0 ? `flex flex-row items-center gap-x-1` : undefined}
+                >
+                  <span
+                    className={
+                      tables.length > 0
+                        ? `text-foreground`
+                        : `text-foreground-muted flex flex-row items-center gap-x-1`
+                    }
+                  >
+                    {displaySchema}
+                    {tables.length === 0 && <Info size={12} />}
+                  </span>
+                </TooltipTrigger>
+                <TooltipContent>
+                  <p>Target schema{tables.length === 0 && ' that will be created'}</p>
+                </TooltipContent>
+              </Tooltip>
+            </TooltipProvider>
+          )}
+        </CardTitle>
```

Comment:
This title has different colour and tooltip treatment depending on the following factors:
- Does the target schema already exist?
- Do tables already exist?
- Is the source a Postgres table or Iceberg client?